conn_send_query.go

```diff
@@ -24,7 +24,7 @@ import (
 // Connection::sendQuery
 // https://github.com/ClickHouse/ClickHouse/blob/master/src/Client/Connection.cpp
 func (c *connect) sendQuery(body string, o *QueryOptions) error {
-	c.debugf("[send query] compression=%t %s", c.compression, body)
+	c.debugf("[send query] compression=%q %s", c.compression, body)
```

Comment:
```suggestion
	c.debugf("[send query] compression=0x%X %s", c.compression, body)
```
It's minor, but perhaps simple notation works better here

Reply:
I think that quoted string like "ztsd" is  clear than hex 0x7A737464, the latter may be confusing when reading log